packages/amesos/src/CMakeLists.txt

```diff
@@ -43,8 +43,6 @@ SET(SOURCES ${SOURCES}
 #
 
 IF (${PACKAGE_NAME}_ENABLE_DSCPACK)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
```

Comment:
Yes, these are clearly redundant as there is already:

* https://github.com/trilinos/Trilinos/blob/2ea9021ea2cdf1f44c3ca84285f4cd8cb6948bc3/packages/amesos/src/CMakeLists.txt#L33